.vsts.pipelines/builds/matrix.yml

```diff
@@ -72,3 +72,14 @@ jobs:
     scriptPrefix: ./
     scriptSuffix: .sh
     setupMac: true
+
+- template: ../jobs/prepare-artifacts.yml
```

Comment:
Should probably condition this to not run on public PR validation actually. (In all other scenarios we may take the previously-source-tarball, but not this.)